client/js/components/comments/comment-box.jsx

```diff
@@ -55,7 +55,11 @@ export default class CommentBox extends React.Component {
           content={this.state.content}
           onSubmit={this._handleFormSubmit} />
         <CommentList comments={this.state.comments}/>
-        <div style={{cursor: 'pointer'}} onClick={this._loadMoreComments}>Load more comments...</div>
+        {() => {
+          if (this.state.comments === 5) {
```

Comment:
For now it loads last 5 comments or all of them when button is clicked, loading paginated comments can be added later.